octobot/src/server/github_handler.rs

```diff
@@ -406,6 +407,25 @@ impl GithubEventHandler {
         let mut participants = pull_request.assignees();
         // add the author of the PR
         participants.push(pull_request.user().clone());
+        // add team members
+        let teams = pull_request.teams();
+        let futures = teams.iter().map(|t| {
```

Comment:
maybe a bit clearer:
- describes what is in the future
- avoids confusing `teams` here with a different `teams` below of a different type.

```suggestion
        let team_members = pull_request.teams().iter().map(|t| {
```

Reply:
I'll make the variable names more distinct, but turns out I still need to assign `pull_request.teams()` into a variable to prevent `temporary value dropped while borrowed` error.